geometry_msgs/Point min
geometry_msgs/Point max
bool densify
---
bool success
